### DISCUSSION TITLE
What's the fastest combo

### DISCUSSION THREAD

**Discussion by meilleur102** [Bodmer/TFT_eSPI#2562] [General]
Hi! 
I'm playing with this library for a while now and I wanted to know what's the fastest combo I can have (display / MCU wise) 
I'm making a display for a Racing simulator steering wheel. 
Since I need to have the Lap time it's in milliseconds. 
I find the ESP32 / ILLI9488 a little to slow when I start asking a lot of information. 
I don't know if it's the Display or the ESP that is the bottleneck here.

I want to try the STM32 (bluepill) but the only one that I have don't work with the StM bootloader. It's only work with the STMDuino bootloader and I saw a post here that this library is not supported. 
When I try to run the code I have a SPI error. 
I need a 4.0inch display. 
Thanks,

**Comment by Bodmer** [comment]
For interface speed the fastest is the RP2040 with 16 bit parallel interface. However the processor does not have an FPU so floating point number crunching is slower. The RP2040 supports DMA for SPI, plus 8 and 16 bit parallel.

For processor speed and for maths intensive programs the Teensy 4.x is fast but only SPI interface is supported.

**Comment by meilleur102** [comment]
Ok thanks. 
I have a ILI9488 with 14 pin so i guest I can't connect it in parallel. I need a display more like the one labeled for Mega with something like 32 pin?
Ill try to find a 4inch parallel display to make some test with the ESP and if it's still not enought ill go with the RP2040.
Do you have a recommendation for a 4inch parallel?
